src/test/java/com/example/pupbuddy/PupBuddyApplicationTests.java

```diff
@@ -90,7 +93,6 @@ private void thenReturnErrorMessageDuplicateTask() {
 
     /**
      * Validate that the DTO properties can be set and retrieved for each class
-     * Thanks to Mahesh Gowda
```

Comment:
Deleted comment giving credit per [previous review](https://github.com/Yep-Was-Taken/pupBuddy/pull/21), since that should be visible in the Git commit history.